src/js/components/Select/__tests__/Select-test.js

```diff
@@ -201,6 +201,24 @@ describe('Select', () => {
     expect(window.scrollTo).toBeCalled();
   });
 
+  test('search with search value prop', () => {
+    const onSearch = jest.fn();
+    const { getByPlaceholderText, getByTestId } = render(
+      <Select
+        id="test-select"
+        options={['one', 'two']}
+        onSearch={onSearch}
+        searchValue="two"
+        searchPlaceholder="search-placeholder"
+        data-testid="test"
+      />,
+    );
+    fireEvent.click(getByTestId('test'));
+
+    const searchInput = getByPlaceholderText('search-placeholder');
+    expect(searchInput.value).toBe('two');
+  });
```

Comment:
I think a better test would be to actually change the value of `searchValue` outside of the render function and then expect is toBe the new value and check that the items in the dropdown match what we are expecting to see from the `searchValue`